tests/utils/test_databricks_utils.py

```diff
@@ -200,3 +202,15 @@ def test_databricks_params_throws_errors(ProfileConfigProvider):
     ProfileConfigProvider.return_value = mock_provider
     with pytest.raises(Exception):
         databricks_utils.get_databricks_host_creds()
+
+
+def test_is_in_databricks_runtime(tmpdir):
+    with mock.patch("sys.modules", new={**sys.modules, "pyspark.databricks": mock.MagicMock()}):
+        # pylint: disable=unused-import,import-error,no-name-in-module
+        import pyspark.databricks  # noqa
+
+        assert databricks_utils.is_in_databricks_runtime()
+    with pytest.raises(ModuleNotFoundError):
```

Comment:
```suggestion
    with pytest.raises(ModuleNotFoundError, match="No module named 'pyspark.databricks'"):
```

Can we add `match` to guarantee that `ModuleNotFoundError` is thrown for `pyspark.databricks`?